### PR TITLE
Katya 

### PR DESCRIPTION
**Pull request by nyotaL**
the base and tests

### CODE REVIEW

**Comment by jeadorf** [comment]
You need to resolve conflicts.

File: src/main/java/com/google/sps/data/Input.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
```
Comment (jeadorf) [inline src/main/java/com/google/sps/data/Input.java:14]: Todo list?

File: src/main/java/com/google/sps/servlets/MailServlet.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      attachment.setFileName("manual.pdf");
+      attachment.setContent(attachmentDataStream, "application/pdf");
+      mp.addBodyPart(attachment);*/
+
+      msg.setContent(mp);
+      // [END multipart_example]
+
+      Transport.send(msg);
+
+    } catch (AddressException e) {
```
Comment (jeadorf) [inline src/main/java/com/google/sps/servlets/MailServlet.java:118]: Exception handling is missing; it's important also to handle the situations where things don't go the way as expected.

**Review by jeadorf — CHANGES_REQUESTED** [review]

**Comment by nyotaL** [comment]
I will pull request once more from my google account

**Pull request closed without merging**